src/DankCord/objects.py

```diff
@@ -343,7 +400,25 @@ def postmemes(description: str):
         else:
             success = False
         return CommandResult(success, death, gain)
-    
+
+    def buy(description: str, item: str, quantity: int):
+        regex = ["\*\*(.*?)\*\*", "[0-9]+"]
```

Comment:
probably compile this regex and then keep calling it when required

Reply:
Should be made better by splitting by constants, this isn't 100% reliable right now.